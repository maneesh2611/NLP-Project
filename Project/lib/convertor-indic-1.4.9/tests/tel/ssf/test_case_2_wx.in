<Sentence id="1">
1	kamitI	NN	<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,o,ti,ti'>|<fs af='kamitI,n,,sg,,o,ti,ti'>
2	kAryanirvahaNaku	NN	<fs af='kAryanirvahaNa,n,,sg,,,ki,ki'>
3	avasaramEna	JJ	<fs af='avasaramavvu,v,any,any,any,,ina,ina'>|<fs af='avasaraM,n,,sg,,,Ena,Ena'>
4	mOlika	NN	<fs af='mOlikaM,n,,sg,,o,ti,ti'>
5	erpAtlapE	NN	<fs af='erpAtu,n,,pl,,,pE,pE'>|<fs af='erpAtu,n,,pl,,,pE,pE'>
6	I	DEM	<fs af='I,avy,,,,,,'>
7	paryatanalo	NN	<fs af='paryatana,n,,pl,,,0,0_o'>|<fs af='paryatana,n,,sg,,,lo,lo'>
8	BAgaMgA	RB	<fs af='BAgaM,n,,sg,,,gA,gA'>
9	wAmu	PRP	<fs af='wAmu,pn,any,sg,2,d,0,0'>
10	samIkRiMcukuMtAmannAru	VM	<fs af='samIkRiMcukuMtAmannAunk,unk,,,,,,'>
11	.	SYM	<fs af='.,punc,,,,,,'>
</Sentence>
<Sentence id="2">
</Sentence>
