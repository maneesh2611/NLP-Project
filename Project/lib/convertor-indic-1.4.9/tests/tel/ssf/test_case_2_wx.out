<Sentence id="1">			
1	కమిటీ	NN	<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>
2	కార్యనిర్వహణకు	NN	<fs af='కార్యనిర్వహణ,n,,sg,,,కి,ki'>
3	అవసరమైన	JJ	<fs af='అవసరమవ్వు,v,any,any,any,,ఇన,ina'>|<fs af='అవసరం,n,,sg,,,ఐన,Ena'>
4	మౌలిక	NN	<fs af='మౌలికం,n,,sg,,o,టి,ti'>
5	ఏర్పాట్లపై	NN	<fs af='ఏర్పాటు,n,,pl,,,పై,pE'>|<fs af='ఏర్పాటు,n,,pl,,,పై,pE'>
6	ఈ	DEM	<fs af='ఈ,avy,,,,,,'>
7	పర్యటనలో	NN	<fs af='పర్యటన,n,,pl,,,0,0_o'>|<fs af='పర్యటన,n,,sg,,,లో,lo'>
8	భాగంగా	RB	<fs af='భాగం,n,,sg,,,గా,gA'>
9	తాము	PRP	<fs af='తాము,pn,any,sg,2,d,0,0'>
10	సమీక్షించుకుంటామన్నారు	VM	<fs af='సమీక్షించుకుంటామన్నాఉన్క్,unk,,,,,,'>
11	।	SYM	<fs af='।,punc,,,,,,'>
</Sentence>			
